games-strategy/warmux/warmux-11.04.1-r2.ebuild

```diff
@@ -13,24 +13,28 @@ S="${WORKDIR}"/${PN}-11.04
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
-IUSE="debug nls unicode"
+IUSE="debug unicode"
 
 RDEPEND="
 	dev-libs/libxml2
+	media-libs/libpng:=
 	media-libs/libsdl[joystick,video,X]
 	media-libs/sdl-image[jpeg,png]
 	media-libs/sdl-mixer[vorbis]
 	media-libs/sdl-ttf
 	media-libs/sdl-net
-	media-libs/sdl-gfx
+	media-libs/sdl-gfx:=
 	media-fonts/dejavu
 	net-misc/curl
 	x11-libs/libX11
-	nls? ( virtual/libintl )
-	unicode? ( dev-libs/fribidi )"
-DEPEND="${RDEPEND}
+	virtual/libintl
+	unicode? ( dev-libs/fribidi )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
 	virtual/pkgconfig
-	nls? ( sys-devel/gettext )"
+	sys-devel/gettext
```

Comment:
Can sort deps here and above.

Reply:
Yes. I tried to, but got confused with the order of letters in the alphabet.

Reply:
Personally I don't really think about it, in vim I just e.g. `:.,30!sort` to sort the range, I tend to prefer POSIX sorting too (capital X comes before a).

May want to do something similar in whichever editor you use.

Reply:
Oh and by "and above" I was implying RDEPEND isn't sorted either.. but well, I'll fix that on merge :)

Reply:
I fixed one line in RDEPEND (which is above), but didn't notice the sdl* ones :(
Using the editor is a good trick, thanks